concrete/src/Updater/Migrations/Migrations/Version20170316000000.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Concrete\Core\Updater\Migrations\Migrations;
+
+use Concrete\Core\Express\EntryList;
+use Concrete\Core\Updater\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20170316000000 extends AbstractMigration
+{
+    public function up(Schema $schema)
```

Comment:
Some comments in here would be nice to make it easy for someone to parse what's happening here. You could also make this a lot easier to read by breaking out the separate parts of this method into other methods.